docs/installation/tks/Installing_TKS_Clone.md

```diff
@@ -0,0 +1,92 @@
+Installing TKS Clone
+====================
+
+Overview
+--------
+
+This page describes the process to install a TKS subsystem as a clone of an existing TKS subsystem.
+
+Exporting Existing System Certificates
+--------------------------------------
+
+Export the existing system certificates (including the certificate chain) into a PKCS #12 file, for example:
+
+```
+$ pki-server tks-clone-prepare --pkcs12-file tks-certs.p12 --pkcs12-password Secret.123
+```
+
+If necessary, third-party certificates (e.g. trust anchors) can be added into the same PKCS #12 file with the following command:
+
+```
+$ pki -d /etc/pki/pki-tomcat/alias -f /etc/pki/pki-tomcat/password.conf \
+    pkcs12-cert-import <nickname> --pkcs12-file tks-certs.p12 --pkcs12-password Secret.123 --append
+```
+
+TKS Subsystem Installation
+--------------------------
+
+Prepare a file (e.g. tks-clone.cfg) that contains the deployment configuration.
+
+A sample deployment configuration is available at [/usr/share/pki/server/examples/installation/tks-clone.cfg](../../../base/server/examples/installation/tks-clone.cfg).
+It assumes that the primary CA and TKS are running at https://primary.example.com:8443,
+the CA signing certificate has been exported into `ca_signing.crt`,
+the admin certificate and key have been exported into `ca_admin_cert.p12`,
+and the admin PKCS #12 password file has been exported into `pkcs12_password.conf`.
+
```

Comment:
okay to assume files names, but where are they expected to be located?

Reply:
Those files are generated from CA installation. Let me update the CA installation doc as well.

Reply:
Please take a look at the updated docs above. Thanks!